cleanowners.py

```diff
@@ -31,6 +42,11 @@ def main():  # pragma: no cover
     codeowners_count = 0
     users_count = 0
 
+    gh_org = get_org(github_connection, organization)
+    if not gh_org:
+        print(f"Organization {organization} not found")
+        return
```

Comment:
Let me know if I'm reading this incorrectly, but wouldn't this if statement end the program if organization is not set and therefor gh_org returns null?

Reply:
Although the test that passes in only a repository list and a organization=None passes so maybe that means I'm wrong here. 🤷🏻

Reply:
I'll double check it. Thank you for asking.

Reply:
@zkoppert yes, it would.  I was being presumptuous.  

The combinatorics on this based on the code:

- (ORGANIZATION and REPOSITORY provided) or (ORGANIZATION not provided and REPOSITORY provided)

  - the repository list from REPOSITORY is used

- ORGANIZATION provided and REPOSITORY not provide

  - we use repositories related to the ORAGANIZATION

- ORGANIZATION not provided and REPOSITORY not provided

  - throw error telling user

In situation 2, if the ORGANIZATION proves to not be an actual ORGANIZATION, we should exit the app, correct?

working on a coding change.  Great catch @zkoppert

Reply:
pushed up new change to do better checking (I hope).  Added new test for the `get_org` function, when org invalid.  Also added black linter to `make lint` 😄

Reply:
Ok, Your summary above makes sense and is how we want it to work. I'll take a peek at the code now.